.vscode/launch.json

```diff
@@ -0,0 +1,116 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
```

Comment:
I know we have mentioned this but for the sake of consistency update .gitignore

Reply:
:+1: